melos.yaml

```diff
@@ -194,7 +194,7 @@ scripts:
       melos run build:integration:firebase --no-select && \
       melos run build:integration:leanplum --no-select && \
       melos bootstrap
-    description: Build the Leanplum integration plugin.
+    description: Build all the integrations.
```

Comment:
teh select package in next line seems also unrelated please remove it

Reply:
removed it as well